### PR TITLE
Remove results from the code base

### PR DESCRIPTION
**Pull request by matthias-ronge**
I have now (hopefully) renamed all the `result`s that we talked about. I hope you like it. I always like it when procedures in the existing code are then implemented in the same way.

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Course.java [hunk 1]
```diff
@@ -382,14 +382,14 @@ public LocalDate getFirstAppearance() {
         if (super.isEmpty()) {
             return null;
         }
-        LocalDate result = super.get(0).getFirstAppearance();
+        LocalDate firstAppearance = super.get(0).getFirstAppearance();
         for (int index = 1; index < super.size(); index++) {
             LocalDate firstAppearance = super.get(index).getFirstAppearance();
```
Comment (oliver-stoehr) [inline Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Course.java]: This cannot compile: `firstAppearance` is already defined.

File: Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Course.java [hunk 2]
```diff
@@ -402,14 +402,14 @@ public LocalDate getLastAppearance() {
         if (super.isEmpty()) {
             return null;
         }
-        LocalDate result = super.get(0).getLastAppearance();
+        LocalDate lastAppearance = super.get(0).getLastAppearance();
         for (int index = 1; index < super.size(); index++) {
             LocalDate lastAppearance = super.get(index).getLastAppearance();
```
Comment (oliver-stoehr) [inline Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Course.java]: This cannot compile: `lastAppearance` is already defined.

File: Kitodo-ImageManagement/src/main/java/org/kitodo/imagemanagement/ImageConverter.java [hunk 3]
```diff
@@ -83,9 +83,9 @@
      * @return the conversion result object to define conversion properties
      */
     FutureDerivative addResult(URI path) {
-        FutureDerivative result = new FutureDerivative(uriToPath(path));
-        results.add(result);
-        return result;
+        FutureDerivative furureDerivative = new FutureDerivative(uriToPath(path));
```
Comment (oliver-stoehr) [inline Kitodo-ImageManagement/src/main/java/org/kitodo/imagemanagement/ImageConverter.java]: There is a typo in `furureDerivative `

**Comment by Kathrin-Huber** [comment]
please resolve conflicts

**Pull request merged**